prisma/schema.sql

```diff
@@ -35,269 +48,366 @@ CREATE TYPE PhysicianStatus as ENUM('ACTIVE', 'INACTIVE');
-  processing_fee                 FLOAT NOT NULL,
-  donation_amount                FLOAT,
-  payment_method                 PaymentType NOT NULL,
-  shopify_confirmation_number    VARCHAR(255) UNIQUE NOT NULL,
-  -- Billing and shipping information
-  shipping_full_name                       VARCHAR(255),
-  shipping_address_line_1                  VARCHAR(255),
-  shipping_address_line_2                  VARCHAR(255),
-  shipping_city                            VARCHAR(255),
-  shipping_province                        Province,
-  shipping_postal_code                     CHAR(6),
-  shipping_address_same_as_home_address    BOOLEAN NOT NULL DEFAULT false,
-  billing_address_same_as_home_address     BOOLEAN NOT NULL DEFAULT false,
-  billing_full_name                        VARCHAR(255),
-  billing_address_line_1                   VARCHAR(255),
-  billing_address_line_2                   VARCHAR(255),
-  billing_city                             VARCHAR(255),
-  billing_province                         Province,
-  billing_postal_code                      CHAR(6),
-  -- Application processing information
-  application_processing_id INTEGER UNIQUE NOT NULL,
+  payment_method PaymentType NOT NULL,
+  processing_fee FLOAT NOT NULL,
+  donation_amount FLOAT,
 
-  created_at              TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  updated_at              TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  -- Shipping information
+  shipping_address_same_as_home_address BOOLEAN NOT NULL,
+  shipping_full_name VARCHAR(255),
+  shipping_address_line_1 VARCHAR(255),
+  shipping_address_line_2 VARCHAR(255),
+  shipping_city VARCHAR(255),
+  shipping_province Province,
+  shipping_country VARCHAR(255),
+  shipping_postal_code VARCHAR(6),
+
+  -- Billing information
+  billing_address_same_as_home_address BOOLEAN NOT NULL,
+  billing_full_name VARCHAR(255),
+  billing_address_line_1 VARCHAR(255),
+  billing_address_line_2 VARCHAR(255),
+  billing_city VARCHAR(255),
+  billing_province Province,
+  billing_country VARCHAR(255),
+  billing_postal_code VARCHAR(6),
+
+  type ApplicationType NOT NULL,
+  notes TEXT,
+  application_processing_id INTEGER UNIQUE NOT NULL,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
 
-  FOREIGN KEY(applicant_id) REFERENCES applicants(id),
   FOREIGN KEY(application_processing_id) REFERENCES application_processing(id)
 );
 
 -- Create permits table
 CREATE TABLE permits (
-  id                SERIAL PRIMARY KEY NOT NULL,
-  rcd_permit_id     INTEGER NOT NULL UNIQUE,
-  expiry_date       DATE NOT NULL,
-  receipt_id        INTEGER,
-  active            BOOLEAN NOT NULL DEFAULT true,
-  application_id    INTEGER NOT NULL UNIQUE,
-  applicant_id      INTEGER NOT NULL,
-  created_at        TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  updated_at        TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  rcd_permit_id INTEGER,
```

Comment:
Should this one also be `SERIAL` or are rcd permit ids assignment differently?

Reply:
`rcd_permit_id` is manually assigned by RCD, so when we create a permit from a completed application, we copy the assigned permit number and use that as the primary key